### PR TITLE
Add tests for prevent duplicate form submissions JS

### PR DESCRIPTION
**Pull request by tombye**
Adds tests for the script that prevents forms being submitted more than once by blocking subsequent clicks on the submit button.

## How to run the tests

`npx jest --config tests/javascripts/jest.config.js tests/javascripts/preventDuplicateFormSubmissions.test.js`

### CODE REVIEW

File: tests/javascripts/preventDuplicateFormSubmissions.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    expect(formSubmitSpy.mock.calls.length).toEqual(1);
+
+  });
+
+  test("It should allow clicks again after 1.5 minutes", () => {
+
+    helpers.triggerEvent(button, 'click');
+
+    jest.advanceTimersByTime(1500);
```
Comment (CrystalPea) [inline tests/javascripts/preventDuplicateFormSubmissions.test.js]: if 1.5 minute is 1500 units, what are those units? :o

Comment (tombye) [reply tests/javascripts/preventDuplicateFormSubmissions.test.js]: Good spot! I've added a comment explaining they're milliseconds. Will update all other tests with this in too.

Comment (CrystalPea) [reply tests/javascripts/preventDuplicateFormSubmissions.test.js]: Thanks so much, today I've learnt ☺️

Comment (CrystalPea) [reply tests/javascripts/preventDuplicateFormSubmissions.test.js]: But isn't 1 second == 1000 milliseconds? And then 1500 milliseconds would be 1.5 second not 1.5 minute... right?

Comment (tombye) [reply tests/javascripts/preventDuplicateFormSubmissions.test.js]: Sorry, missed that completely. I've rebased this to just add a commit fixing the test description as I don't think it needs any more explanation if it's talking about the right units.

Comment (CrystalPea) [reply tests/javascripts/preventDuplicateFormSubmissions.test.js]: Ahh, ok, no worries and thanks, I think that was what bamboozled me at first :D

Comment (CrystalPea) [reply tests/javascripts/preventDuplicateFormSubmissions.test.js]: doggo
